pkg/text/computeacl.go

```diff
@@ -0,0 +1,61 @@
+package text
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/fastly/go-fastly/v9/fastly/computeacls"
+	"github.com/segmentio/textio"
+)
+
+// PrintComputeACL displays a compute ACL.
+func PrintComputeACL(out io.Writer, prefix string, acl *computeacls.ComputeACL) {
+	out = textio.NewPrefixWriter(out, prefix)
+
+	fmt.Fprintf(out, "\nID: %s\n", acl.ComputeACLID)
+	fmt.Fprintf(out, "Name: %s\n", acl.Name)
+}
+
+// PrintComputeACLsTbl displays compute ACLs in a table format.
+func PrintComputeACLsTbl(out io.Writer, acls []computeacls.ComputeACL) {
+	tbl := NewTable(out)
+	tbl.AddHeader("Name", "ID")
+
+	if acls == nil {
+		tbl.Print()
+		return
+	}
+
+	for _, acl := range acls {
+		// avoid gosec loop aliasing check :/
+		acl := acl
+		tbl.AddLine(acl.Name, acl.ComputeACLID)
+	}
+	tbl.Print()
+}
+
+// PrintComputeACLEntry displays a compute ACL entry.
+func PrintComputeACLEntry(out io.Writer, prefix string, entry *computeacls.ComputeACLEntry) {
+	out = textio.NewPrefixWriter(out, prefix)
+
+	fmt.Fprintf(out, "\nPrefix: %s\n", entry.Prefix)
+	fmt.Fprintf(out, "Action: %s", entry.Action)
```

Comment:
Since this string does not contain '\n', `PrefixWriter` won't output it unless `out.Flush()` is called (see https://github.com/segmentio/textio#indenting). If this string can't include a '\n' because it needs to support continuation, then an explicit `Flush()` in this function would be a suitable fix.

Reply:
Agree, and please note that the same applies to all other uses of `PrefixWriter`.

Reply:
I just missed to add the `\n`; addressed in c766bc2af3bc2b7a61f556ce2fac779309c5a6dd.